tests/testthat/testthat_monthlydownscale_functions.R

```diff
@@ -1,39 +1,51 @@
 
 context('monthly downscaling')
 
+## For some reason, setting cores to anything > 1 causes `R CMD check --as-cran`
+## to fail.  Running without the cran setting works with larger core counts.
+## The following test causes the parallel version to be tested when running
+## locally, while still functioning when running under CRAN rules.
+notcran <- Sys.getenv('NOT_CRAN') == 'true'
```

Comment:
Apparently this does not exactly do what I had hoped.  Devtools sets the `NOT_CRAN` environment variable whenever it is running the tests, even if it is running with the `--as-cran` flag.  So, the tests will probably succeed on Travis, or on CRAN itself, but `devtools::check()` still fails (unless you set `cran = FALSE`, but there is no way to get Rstudio to do this).

Reply:
Hmm I don't have any ideas to work around this right now, I am gonna have to think about it.